Guide.docc/MigrationGuide.md

```diff
@@ -14,27 +14,27 @@
 
 Swift's concurrency system, introduced in [Swift 5.5](https://www.swift.org/blog/swift-5.5-released/),
 makes asynchronous and parallel code easier to write and understand.
-With the Swift 6 language mode, the compiler can now also
-guarantee that concurrent programs are data-race-free.
-When this mode is enabled, the compiler safety checks that were
-optional become required.
+With the Swift 6 language mode, the compiler can now 
+guarantee that concurrent programs are free of data races.
+When enabled, compiler safety checks that were
```

Comment:
I think it is clear that Swift 6 language mode from the previous sentence is being referred to by 'When enabled'.